main/app.py

```diff
@@ -0,0 +1,352 @@
+
+        self.living_occupied = 0
+        self.office_occupied = 0
+        file = self.file = " "
+        allocated = self.allocated = []
+
+        self.conn = sqlite3.connect("amity.sqlite")
+        self.conn.row_factory = sqlite3.Row
+        self.connect = self.conn.cursor()
+        self.connect.execute(
+            "CREATE TABLE IF NOT EXISTS Rooms(id INTEGER PRIMARY KEY AUTOINCREMENT, Name TEXT, type TEXT)")
+
+        self.connect.execute(
+            "CREATE TABLE IF NOT EXISTS Persons(id INTEGER PRIMARY KEY AUTOINCREMENT, Name TEXT, Personnel_type TEXT, want_accommodation TEXT, office_accommodation TEXT , living_accomodation TEXT)")
+
+    def create_rooms(self, room):
+        """Allows user to enter a list of room names specifying
+                whether office or living spaces"""
+        room_type = raw_input(
+            "Enter room type: \n O: Office space \n L: Living space: \n")
+
+        while room_type != "O" and room_type != "L":
+            room_type = raw_input(
+                "Try again. Enter Room Type:\n O: Office space \n L: Living space: \n")
+        for room_name in room:
+            if room_type.upper() == 'O':
+                self.rooms[room_type.upper()].append(room_name)
+            else:
+                self.rooms['L'].append(room_name)
+            self.connect.execute(
+                "INSERT INTO Rooms (Name, type) VALUES (?, ?)", [room_name, room_type])
+            self.conn.commit()
+        print('New rooms succesfully created')
+        return room
+
+    def office_space_count(self, off_name):
+        '''Keep track of offices allocated should be less than 6'''
+        off_name = str(off_name)
+        office_space = self.connect.execute(
+            "SELECT COUNT(*) AS office_occupants FROM Persons  WHERE Persons.office_accommodation = ?", [off_name]).fetchall()
+        self.office_occupied = office_space[0][0]
+        return self.office_occupied
+
+    def living_space_count(self, liv_name):
+        '''Keep track of living space allocated should be less than 4'''
+        # print(liv_name)
+        living_space = self.connect.execute(
+            "SELECT COUNT(*) AS living_occupants FROM Persons WHERE Persons.living_accomodation = ?", [liv_name]).fetchall()
+        for liv_space in living_space:
+            self.living_occupied = liv_space[0]
+            print(self.living_occupied)
+        return self.living_occupied
+
+    def add_person(self, first_name, last_name, person_type, want_housing):
+        '''Method receives personnel data and calls other method to save and allocate rooms to personnel '''
+        name = first_name + " " + last_name
+        person_type = person_type
+        want_accommodation = want_housing
+        if person_type.upper() == 'STAFF':
+            self.people_data['Staff'].append(name)
+        else:
+            self.people_data['Fellow'].append({name: want_accommodation})
+
+        self.insert_db(name=name, person_type=person_type,
```

Comment:
**Code Quality: Nitpick**

What happens here in the case where we have a staff. I'm referring to the `wants_accomodation` variable.